jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -363,6 +393,24 @@ def _display_args_changed(self, msg={}):
                 mark.colors = [self.color]
                 mark.fill_opacities = [self.fill_opacity]
 
+    def import_region(self, region):
+        """
+        Import an astropy regions object (or file).
+
+        Parameters
+        ----------
+        region : str or regions.Regions object
+        """
+        if isinstance(region, (regions.Region, regions.Regions)):

```

Comment:
Maybe in the subsequent refactoring (not this PR), we can use the existing regions API from here:

https://github.com/spacetelescope/jdaviz/blob/bbcdcc8a4d463dc4a9eae69b2a01a6567f23539d/jdaviz/core/helpers.py#L616C80-L616C80

and

https://github.com/spacetelescope/jdaviz/blob/bbcdcc8a4d463dc4a9eae69b2a01a6567f23539d/jdaviz/core/helpers.py#L648